docs/src/tuning_models.md

```diff
@@ -173,6 +173,53 @@ The prior distributions used for sampling each hyperparameter can be
 customized, as can the global fallbacks. See the
 [`RandomSearch`](@ref) doc-string below for details.
 
+## Tuning using latin hypercube sampling
+
+We can also tune the hyperparameters using `LatinHypercube`. This 
+method instantiates a grid-based hyperparameter tuning strategy 
+by using a genetic based optimization algorithm based on the 
+inverse of the Audze-Eglais function, using the library 
+`LatinHypercubeSampling.jl`.
+Let's instatiate the tuning strategy:
+```@repl goof
+latin_minimal = LatinHypercube(nGenerations=2,popSize= 120)
+```
+Where nGenerations is the number of generations to run the 
```

Comment:
I'm a bit confused by the definition of `nGenerations` and the other terminology. Presumably, this has nothing to do with the iteration parameter `n` specified in the `TunedModel` construction, but the way it is presented here makes it sound like it *is* the iteration parameter. Also, is it really fair to call this genetic *optimisation*? Aren't we just spitting out the entire set of models to be evaluated in one hit? When I think of [genetic optimisation](https://towardsdatascience.com/genetic-algorithm-to-optimize-machine-learning-hyperparameters-72bd6e2596fc) I think of a sequential algorithm in which the choice of new models to be evaluated at each step depends on the outcomes of previous ones.

Reply:
I'm probably just ignorant about the algorithm, but others are likely to be confused.

Reply:
I see what you mean, after reading again what I have written I see where the confusion comes from. The inner working of the library is actually based on a genetic algorithm, and all the parameters refer to this in one way or the other. So I just made the distinction clearer, let me know if that's better now.